shell/renderer/electron_render_frame_observer.cc

```diff
@@ -60,6 +61,18 @@ void ElectronRenderFrameObserver::DidClearWindowObject() {
   renderer_client_->DidClearWindowObject(render_frame_);
 }
 
+void ElectronRenderFrameObserver::DidCreateScriptContext(
+    v8::Handle<v8::Context> context,
+    int world_id) {
+  // Preload JS doesn't run early enough in context initialization to be able to
+  // modify conditional features because of issues with some preload scripts
+  // expecting JS features that aren't yet enabled to be present when the
+  // preload runs. So we have to enable this feature early.
```

Comment:
I think we could enable this only in isolated world too if we want and require discord_voice to expose it but I didn't see a super necessary reason to do that, wdyt